packages/markdown-render/__test__/__snapshots__/index.test.ts.snap

```diff
@@ -16,14 +16,8 @@ exports[`Empty Components (without slots,methods,props,events) can be forced to
 
 exports[`Proper rendering of the table header 1`] = `
 Object {
-  "events": "|Event Name|Description|Parameters|
-|---|---|---|
-|click|Triggered when clicked|a boolean value|
-",
-  "methods": "|Method|Description|Parameters|
-|---|---|---|
-|clear|Clear form|a boolean value|
-",
+  "events": "<table><thead><tr><th>Event Name</th><th>Description</th><th>Parameters</th><th>Type</th><th>Default</th></tr></thead><tbody><tr><td rowspan=1>click</td><td rowspan=1>Triggered when clicked</td><td rowspan=1>test</td><td rowspan=1>Boolean</td><td rowspan=1>false</td></tr></tbody></table>",
+  "methods": "<table><thead><tr><th>Method</th><th>Description</th><th>Parameters</th><th>Type</th><th>Default</th></tr></thead><tbody><tr><td rowspan=1>clear</td><td rowspan=1>Clear form</td><td rowspan=1>test</td><td rowspan=1>Boolean</td><td rowspan=1>false</td></tr></tbody></table>",
```

Comment:
This was exactly my suggestion in #128 to convert the markdown tables to its html equivalent to format multi-line comments. This PR should resolve my issue as well. Hopefully the author comes back and accepts this, this is a great PR.